### PR TITLE
Set player type

### PR DESCRIPTION
**Pull request by ypavlotsky**
Updated IMA SDK to v3.beta13.3. 
Added call to setPlayerType to identify GMF player. 
Updated Google Play Services version.

### CODE REVIEW

File: demo/src/main/java/com/google/googlemediaframeworkdemo/demo/adplayer/ImaPlayer.java [hunk 1]
```diff
@@ -55,6 +55,8 @@
  */
 public class ImaPlayer {
 
+  private static String PLAYER_TYPE = "GMF ImaPlayer";
```
Comment (shawnbuso) [inline demo/src/main/java/com/google/googlemediaframeworkdemo/demo/adplayer/ImaPlayer.java]: Can we change this to something that indicates that this is Android and not iOS? Maybe "gmf_android" or something like that.

File: demo/src/main/java/com/google/googlemediaframeworkdemo/demo/adplayer/ImaPlayer.java [hunk 2]
```diff
@@ -297,7 +299,7 @@ public void removeCallback(VideoAdPlayerCallback videoAdPlayerCallback) {
      * then the start method will be ignored.
      */
     @Override
-    public VideoProgressUpdate getProgress() {
+    public VideoProgressUpdate getAdProgress() {
```
Comment (shawnbuso) [inline demo/src/main/java/com/google/googlemediaframeworkdemo/demo/adplayer/ImaPlayer.java]: Is this an IMA API change?

**Comment by shawnbuso** [comment]
Added comments, PTAL.

File: demo/src/main/java/com/google/googlemediaframeworkdemo/demo/adplayer/ImaPlayer.java (hunk 2 above)
Comment (ypavlotsky) [inline demo/src/main/java/com/google/googlemediaframeworkdemo/demo/adplayer/ImaPlayer.java]: Yes, it is. VideoAdPlayer now extends AdProgressProvider which provides the newly-named method and the old method is gone.

File: demo/src/main/java/com/google/googlemediaframeworkdemo/demo/adplayer/ImaPlayer.java (hunk 1 above)
Comment (tensafefrogs) [inline demo/src/main/java/com/google/googlemediaframeworkdemo/demo/adplayer/ImaPlayer.java]: The example code uses "'google/gmf-player'" as the id. I would advocate for google/gmf-android and you should also set the version if possible.

**Comment by tensafefrogs** [comment]
Please also squash your two commits into a single commit.

**Comment by tensafefrogs** [comment]
Also the Travis build failed due to "Could not find com.google.android.gms:play-services-ads:8.1.0."

**Comment by ypavlotsky** [comment]
Squashed my commits.

**Comment by tensafefrogs** [comment]
LGTM

**Pull request merged**